ajax/libs/redux-persist/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "redux-persist",
+  "version": "4.0.0-alpha6",
+  "description": "persist and rehydrate redux stores",
+  "main": "lib/index.js",
```

Comment:
Sorry, I use the importer tool generate the package.json and forgot to run npm test. I will correct it soon